wear/src/main/java/chat/rocket/android/util/DrawableHelper.kt

```diff
@@ -0,0 +1,61 @@
+package chat.rocket.android.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.DrawableCompat
+import chat.rocket.android.R
+import chat.rocket.common.model.UserStatus
+
+object DrawableHelper {
```

Comment:
You don't need to duplicate `DrawableHelper`. You can put this class in the `util` module and use it in the `wear` module and app module as well.